contrib/lua/premake5.lua

```diff
@@ -21,8 +21,8 @@ project "lua-lib"
 		"etc/*.c"
 	}
 
-	configuration "linux or bsd or hurd or aix"
+	filter "system:linux or system:bsd or system:hurd or system:aix"
```

Comment:
I'm still impressed that a single line (and suggestion) started quite a discussion worth of a separate "issue"...

And yeah for the longest time I didn't even know that "and" is actually possible because I only ever saw "or" but never "and" (I mean this syntax was also present with "configuration" so it's been there for quite a while)